### PR TITLE
Added a spinner when loading plus made images and divs constant size

### PR DESCRIPTION
**Pull request by jotjern**

### CODE REVIEW

**Review by jotjern — COMMENTED** [review]
doesn't lgtm

File: src/App.tsx
Comment (NjaalSoerland) [inline src/App.tsx:1]: think this should be handled in a more dynamic manner

**Review by NjaalSoerland — CHANGES_REQUESTED** [review]

File: src/App.tsx
Comment (jotjern) [reply src/App.tsx:1]: I don't

Comment (NjaalSoerland) [reply src/App.tsx:1]: too bad

**Pull request closed without merging**